src/Uno.UWP/Generated/3.0.0.0/Windows.Devices.Sensors/LightSensor.cs

```diff
@@ -2,12 +2,12 @@
 #pragma warning disable 114 // new keyword hiding
 namespace Windows.Devices.Sensors
 {
-	#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
+	#if false
```

Comment:
Why false for all, and not only for implemented? (WASM/Android)?

Reply:
Because the class itself is available everywhere and its `GetDefault()` method will return `null` which is in line with how other sensors behave in Uno and in line with UWP when the sensor is not supported. This way you can write the code as "fully cross platform" without having to add #if checks, because `GetDefault()` will always return `null` and not throw.